src/ontology/uberon.Makefile

```diff
@@ -1233,55 +1233,41 @@ $(MAPPINGDIR)/uberon-local.sssom.tsv: $(SRC) | all_robot_plugins
-				$(MAPPINGDIR)/zfa.sssom.tsv
+				$(MAPPINGDIR)/fbbt.sssom.tsv
 	sssom-cli $(foreach src, $^, -i $(src)) \
 		  --prefix-map-from-input \
-		  --rule '(object==UBERON:* || object==CL:*) -> invert()' \
-		  --rule '(subject==UBERON:* || subject==CL:*) -> include()' \
+		  --rule 'object==UBERON:* -> invert()' \
+		  --rule 'subject==UBERON:* -> include()' \
 		  --output $@
 
 # Remote mapping sets
 # -------------------
 
 # The following providers publish their own mapping sets.
-EXTERNAL_SSSOM_PROVIDERS = fbbt cl zfa biomappings
+EXTERNAL_SSSOM_PROVIDERS = fbbt cl biomappings
 
 # All the sets coming from the above ontologies.
 EXTERNAL_SSSOM_SETS = $(foreach provider, $(EXTERNAL_SSSOM_PROVIDERS), $(MAPPINGDIR)/$(provider).sssom.tsv)
 
 # We only refresh external resources under IMP=true
 ifeq ($(strip $(IMP)),true)
 
-# FBbt mapping set. Nominally a simple mirror, but we need a custom rule
-# because the default, ODK-generated rule would ignore the MIR variable.
+# FBbt mapping set. We filter it to only keep UBERON-related mappings
+# (CL-related mappings are already bundled with the CL set).
 $(MAPPINGDIR)/fbbt.sssom.tsv: .FORCE
-	wget "http://purl.obolibrary.org/obo/fbbt/fbbt.sssom.tsv" -O $@
-
-# CL mapping set (extracted from CL cross-references). We need to first
-# merge Uberon with CL, because the treat-xrefs-as-... annotations are
-# only in Uberon; then we need to remove the Uberon terms to avoid
-# extracting the cross-references from them as well.
-$(MAPPINGDIR)/cl.sssom.tsv: $(SRC) $(IMPORTDIR)/local-cl.owl | all_robot_plugins
-	$(ROBOT) merge -i $(SRC) -i $(IMPORTDIR)/local-cl.owl --collapse-import-closure false \
-		 remove --base-iri http://purl.obolibrary.org/obo/CL_ --axioms external \
-		 sssom:xref-extract --mapping-file $@ -v --drop-duplicates \
-		                    --set-id "$(ONTBASE)/mappings/cl-mappings.sssom.tsv" \
-		                    --prefix 'KUPO:  http://purl.obolibrary.org/obo/KUPO_'  \
-		                    --prefix 'SCTID: http://purl.obolibrary.org/obo/SCTID_' \
-		                    --prefix 'FMA:   http://purl.org/sig/ont/fma/fma' \
-	> $(REPORTDIR)/cl-xrefs-extraction.txt
+	wget "http://purl.obolibrary.org/obo/fbbt/fbbt.sssom.tsv" -O - | \
+		sssom-cli --prefix-map-from-input \
+		          --rule 'object==UBERON:* -> include()' \
+		          --output $@
 
-# ZFA mapping set (extracted from ZFA cross-references).
-$(MAPPINGDIR)/zfa.sssom.tsv: $(IMPORTDIR)/local-zfa.owl | all_robot_plugins
-	$(ROBOT) sssom:xref-extract -i $(IMPORTDIR)/local-zfa.owl --mapping-file $@ \
-		                    -v --drop-duplicates \
-		                    --set-id "$(ONTBASE)/mappings/zfa-mappings.sssom.tsv" \
-	> $(REPORTDIR)/zfa-xrefs-extraction.txt
+# CL mapping set. We simply fetch it as it is from CL. It already
+# contains all mappings related to CL (including the FBbt and ZFA ones).
+$(MAPPINGDIR)/cl.sssom.tsv: .FORCE
+	wget "http://purl.obolibrary.org/obo/cl/cl.sssom.tsv" -O $@
 
```

Comment:
Should this be under the MIR false regime?

Reply:
The entire section is already within a conditional block:

```Make
ifeq ($(strip $(IMP)),true)
    ...
endif
```

It’s dependent on `IMP` rather than `MIR` because we are not _mirroring_ anything here. Contrary to imports, for which we have a two-steps process (first mirroring, and then creating the import module proper), for foreign mappings it’s all done in a single step.